.github/workflows/lint.yaml

```
@@ -34,5 +35,5 @@ jobs:
       # Format used:
       # ::error file={filename},line={line},col={col}::{message}
       - name: Run flake8
-        run: "flake8 --format='::error file=%(path)s,line=%(row)d,col=%(col)d::\
-            [flake8] %(code)s: %(text)s'"
+        run: "poetry run flake8 --format='::error file=%(path)s,line=%(row)d,\
+        col=%(col)d::[flake8] %(code)s: %(text)s'"
```

If we disable virtualenvs we don't need to specify poetry run

```suggestion
        run: "flake8 \
        --format='::error file=%(path)s,line=%(row)d,col=%(col)d::\
        [flake8] %(code)s: %(text)s'"
```